### PR TITLE
[DOC] Fix attributes file title

### PR DESCRIPTION
**Pull request by ziyaointl**
The attributes file has a title of environmet. This PR fixes the typo.

### CODE REVIEW

File: docs/reference/attributes.rst [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ===========
-Environment
+Attributes
 ===========
```
Comment (parsoyaarihant) [inline docs/reference/attributes.rst:3]: I believe you also need to change number of `=` characters to the length of section name (`Attributes`). Else the document will not build. 

Refer: https://thomas-cokelaer.info/tutorials/sphinx/rest_syntax.html#headings

Comment (ziyaointl) [reply docs/reference/attributes.rst:3]: Oops, didn't know this before. Thanks for pointing it out!

**Comment by jeremydouglass** [comment]
@parsoyaarihant -- did you have any more requests for changes on this PR?

**Comment by parsoyaarihant** [comment]
@ziyaointl Thanks!

**Pull request merged**